data/human/human missions.txt

```diff
@@ -2502,6 +2502,123 @@ event "terraforming follow-up"
+			choice
+				`	"Alright. I'll prepare a bunk for you."`
+				`	"Sorry, that's too far out of my way."`
+					decline
+			`	She is delighted, and explains that she will meet you at your ship later today, before leaving to attend to her work in preparation for her absence.`
+				accept
+	on complete
+		conversation
+			`As you approach <origin>, Amy is astounded by the magnitude of volcanic ash in the air, far outweighing what was released by the volcano you triggered on Tundra. "This place makes Tundra seem temperate," Amy remarks after observing the glaciers that occupy most of the planet's polar regions.`
+			`	Her fascination with the state of this world is temporarily forgotten when she exits your ship and sees the Wanderers for the first time. "I don't know what I was expecting them to be like, but it wasn't birds." However, she quickly returns to the reason for coming here. "Do you know how we can find someone who can answer my questions?"`
+			`	You approach one of the nearby Wanderers and ask if they could direct you to Tema'a Iriket, the terraforming director. They don't appear to understand the Hai language, but they seem to recognize the name and gesture for you to wait in the spaceport, before heading off in a different direction, presumably to fetch Iriket for you.`
+
+
+
+mission "Terraforming 14"
+	name "Field Trip"
+	description "Take Tema'a Iriket and Amy to <destination> to show her a world where the Wanderers have finished their work."
+	source "Desi Seledrak"
+	destination "Var' Kayi"
+	passengers 2
+	blocked "You need <capacity> in order to take on this mission."
+	to offer
+		has "Terraforming 13: done"
+	on offer
+		conversation
+			`It is not long before Iriket arrives. You introduce Amy to her, and explain why you have brought her here. "I would be [glad, honored] to answer any questions about our work," she agrees.`
+			`	What follows is an extensive discussion about how the Wanderers go about planning the terraforming process for a world. From the earliest considerations as to what the goal for each planet should be to the role of their artificial minds in helping to monitor and guide the process in the long term. Your translator has some difficulty with the more technical aspects of the discussion, which leads Iriket to rephrase some things many times. Throughout it all, Amy is taking copious notes. Iriket also explains some of what they have been doing to improve the climate on this world.`
+			`	"The gases [released, produced] by the factories will help the atmosphere retain more heat, which will in turn begin to melt the glaciers and return water to the [ecosystem, environment]. Over time, this will [mix, interact] with the abundant ash, helping to [restore, rejuvenate] the soils all over this world. In time, it will become [fruitful, bountiful] again."`
```

Comment:
```suggestion
			`	"The gases [released, produced] by the factories will help the atmosphere retain more heat, which will, in turn, begin to melt the glaciers and return water to the [ecosystem, environment]. Over time, this will [mix, interact] with the abundant ash, helping to [restore, rejuvenate] the soils worldwide. In time, it will become [fruitful, bountiful] again."`
```

Reply:
The final sentence here says "it," referring to "this world." I don't think that works if we say "worldwide."

Reply:
Hmm. I was under the impression that "it" referred to the soil.

Reply:
It could be seen both ways, so it might be nice to change it slightly.

Reply:
Given that both work and have essentially the same meaning, I don't know what change would be necessary.